# 
# Let's default to ARCH=powerpc, even in 32bit.
# Author: David Woodhouse <david@woodhou.se>
# Upstream status: upstream still uses ARCH=ppc on 32bit, but they have to
# deal also with the embedded arches debian doesn't support.
#

--- linux-2.6.14/Makefile.orig	2005-11-19 17:13:52.000000000 +0000
+++ linux-2.6.14/Makefile	2005-11-19 17:14:04.000000000 +0000
@@ -169,7 +169,7 @@ KERNELRELEASE=$(VERSION).$(PATCHLEVEL).$
 SUBARCH := $(shell uname -m | sed -e s/i.86/i386/ -e s/sun4u/sparc64/ \
 				  -e s/arm.*/arm/ -e s/sa110/arm/ \
 				  -e s/s390x/s390/ -e s/parisc64/parisc/ \
-				  -e s/ppc64/powerpc/ )
+				  -e s/ppc.*/powerpc/ )
 
 # Cross compiling and selecting different set of gcc/bin-utils
 # ---------------------------------------------------------------------------
